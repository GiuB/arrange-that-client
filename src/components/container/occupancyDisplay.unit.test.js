import React from 'react';
import Enzyme, {render} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import toJson from 'enzyme-to-json';

import OccupancyDisplay from './occupancyDisplay';

Enzyme.configure({adapter: new Adapter()});

it('Shows default occupancy display', () => {
    const occupancy = render(<OccupancyDisplay count={7} total={8} />);

    expect(toJson(occupancy, {})).toMatchSnapshot();
});

it('Shows over occupancy display', () => {
    const overOccupancy = render(<OccupancyDisplay count={9} total={8} />);

    expect(toJson(overOccupancy, {})).toMatchSnapshot();
});