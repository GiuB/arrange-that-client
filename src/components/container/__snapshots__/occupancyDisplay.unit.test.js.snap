// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Shows default occupancy display 1`] = `
<div>
  <div
    class="MuiChip-root-3 OccupancyDisplay-chip-2"
    role="button"
    tabindex="-1"
  >
    <span
      class="MuiChip-label-22"
    >
      7/8
    </span>
  </div>
</div>
`;

exports[`Shows over occupancy display 1`] = `
<div>
  <div
    class="MuiChip-root-3 MuiChip-colorSecondary-5 OccupancyDisplay-chip-2"
    role="button"
    tabindex="-1"
    title="Over the size limit"
  >
    <span
      class="MuiChip-label-22"
    >
      <svg
        aria-hidden="true"
        class="MuiSvgIcon-root-36"
        focusable="false"
        role="presentation"
        viewBox="0 0 24 24"
      >
        <path
          d="M0 0h24v24H0z"
          fill="none"
        />
        <path
          d="M1 21h22L12 2 1 21zm12-3h-2v-2h2v2zm0-4h-2v-4h2v4z"
        />
      </svg>
    </span>
  </div>
</div>
`;
